tools/scripts/roundtrip_test.sh

```diff
@@ -116,9 +116,9 @@ main() {
   roundtrip_test "jxl/flower/flower_small.rgb.png" "-e 1" 0.02
   roundtrip_test "jxl/flower/flower_small.rgb.png" "-e 1 -d 0.0" 0.0
   roundtrip_test "jxl/flower/flower_small.rgb.depth8.ppm" \
-		 "-e 1 --streaming_input" 0.02
+		 "-e 1 --streaming_input --streaming_output" 0.02
```

Comment:
Might make sense to test things separately

Reply:
Done. And it caught a bug, which you probably noticed but it was probably the easiest way to point out :)

Reply:
I did not notice a bug :P what was it?

Reply:
JxlEncoderFlushInput was not called (and it called automatically for streaming input+output)

Reply:
Ah, indeed that would be a difference...